ext/LogDensityProblemsADADTypesExt.jl

```diff
@@ -9,7 +9,7 @@ else
 end
 
 """
-    ADgradient(ad::ADTypes.AbstractADType, ℓ)
+    ADgradient(ad::ADTypes.AbstractADType, ℓ; x=nothing)
```

Comment:
Maybe document the type constraint?
```suggestion
    ADgradient(ad::ADTypes.AbstractADType, ℓ; x::Union{Nothing,AbstractVector}=nothing)
```